packages/features/pbac/PBAC_REFACTORING_GUIDE.md

```
@@ -24,63 +27,262 @@ const teamsToQuery = (
+   const permissionCheckService = new PermissionCheckService();
+   const teamIdsWithPermission = await permissionCheckService.getTeamIdsWithPermission(
+     session.user.id,
+     "team.listMembers"
+   );
+   ```
+
+const permissions = {
+canListMembers: teamIdsWithPermission.length > 0 // Permission-specific name
+};
+
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)